### PR TITLE
Some changes in creating, updating channels page.

### PR DESCRIPTION
**Pull request by joe513**

### CODE REVIEW

File: templates/app/channel_form.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 {% block content %}
   {% if object.path %}
+    <div class="container" align="center">
```
Comment (inoks) [inline templates/app/channel_form.html]: `align="center"` is it correct attribute?

File: templates/app/channel_form.html [hunk 2]
```diff
@@ -30,10 +31,16 @@
...
-  <form method="POST">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <button>{% trans "Save" %}</button>
-  </form>
+  </div>
+  <div class="container">
+    <form method="POST"> {% csrf_token %}
 
+      {% for field in form %}
```
Comment (inoks) [inline templates/app/channel_form.html]: What is the purpose of it?

File: app/forms.py [hunk 3]
```diff
@@ -4,26 +4,24 @@
...
-        model = Channel
-        fields = [
-            'title',
-            'path',
-            'group',
-            'hidden',
-        ]
+        widgets = {
+            'path': forms.Textarea({'cols': 50, 'rows': 1}),
+            'extra_data': forms.Textarea({'cols': 50, 'rows': 5})
```
Comment (inoks) [inline app/forms.py]: User should not manually edit JSON in order to update values, we need to have simple inputs for each editable value in the  `extra_data`.

File: templates/app/channel_form.html (hunk 1 above)
Comment (joe513) [reply templates/app/channel_form.html]: @inoks seems like [yes](https://www.w3schools.com/tags/tag_div.asp)  I can delete it, it doesn't matter a lot for us

File: templates/app/channel_form.html (hunk 2 above)
Comment (joe513) [reply templates/app/channel_form.html]: @inoks I edited this to be able to add `<br>`, `<hr/>` tags.

**Pull request closed without merging**